kde/kate/Portfile

```diff
@@ -15,25 +15,26 @@ long_description    The Kate project develops two main products: KatePart, the a
                     a text editing component, and Kate, a MDI text editor application. \
                     In addition, we provide KWrite, a simple SDI editor shell which allows \
                     the user to select his/her favourite editor component.
-platforms           darwin
-homepage            http://kate-editor.org/
+homepage            https://kate-editor.org
 master_sites        kde:stable/${version}/src/
 use_xz              yes
 distname            kate-${version}
 checksums           rmd160  195cfac39111a1434b6b5c60543b219a70c0c74e \
-                    sha256  513d4ae2c36fa6b59caf3b2b685e3ea1167093d16025859cfac90d75617e707d
+                    sha256  513d4ae2c36fa6b59caf3b2b685e3ea1167093d16025859cfac90d75617e707d \
+                    size    2766880
 
-depends_lib-append  port:kdelibs4 \
-                    port:qjson \
-                    port:kactivities \
-                    port:oxygen-icons
+depends_lib-append  port:kactivities \
+                    port:kdelibs4 \
+                    port:oxygen-icons \
+                    port:qjson
 
 #Avoids weird behaviour with Zoom on OS X (https://git.reviewboard.kde.org/r/120319/)
 patchfiles          patch-Zoom.diff \
                     patch-kateviewhelpers.diff
 
-#Blacklist gcc42 and llvm-gcc-42 (does not build with gcc, ticket #37596)
-compiler.blacklist  gcc-4.2 apple-gcc-4.2 llvm-gcc-4.2 macports-llvm-gcc-4.2
+#Blacklist gcc42 and llvm-gcc-42
```

Comment:
So the effect of this change is to additionally blacklist `*gcc-4.0`, yes? Then the comment isn't accurate anymore. The code is self-explanatory as to what it does so I'd remove that part of the comment. I'd keep the reference to the bug report that explains why the code is needed.
```suggestion
# https://trac.macports.org/ticket/37596
```
The ticket was about an inability to link when using gcc 4.2. If you're now saying that gcc 4.0 can't link it either, it seems unlikely that earlier versions would be able to link it. Should `gcc-3.3` be added to the list?

Reply:
@ryandesign If the presumption should be that blacklist was correct and gcc-4.0 was working while gcc-4.2 failed (on whatever system it was tried on), then I can check if gcc-4.0 succeeds.
These ports take quite some time to build even on the fast G5, so I assumed that gcc-4.0 was omitted because not being picked anyway wherever tested earlier. It is surprising if indeed 4.2 fails where 4.0 works, and make 4.2 does not fail after all either?

Anyway, if we see such possibilities, let me verify with both.